frontend/lib/repository/net/https.dart

```diff
@@ -0,0 +1,152 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:flutter/foundation.dart' show kIsWeb;
+import '../types/types.dart';
+import 'package:http/http.dart' as http;
+
+// //kIsWeb documentation: https://api.flutter.dev/flutter/foundation/kIsWeb-constant.html
+
+// Todo: Create class for network stuff
+
+String url;
+// String text = "";
+
+Future<MasterData> createMasterDataResponse() async {
+  print("CALL: util/net.dart createMasterDataResponse()");
+  if (kIsWeb) {
+    url =
+        'https://cors-anywhere.herokuapp.com/ https://europe-west3-serverless-devops-play.cloudfunctions.net/get-master-data';
+  } else {
+    url =
+        'https://europe-west3-serverless-devops-play.cloudfunctions.net/get-master-data';
+  }
+
+  print("net.dart url: " + url);
+
+  final http.Response response = await http.post(
+    url,
+    headers: <String, String>{
+      'Content-Type': 'application/json; charset=UTF-8',
+    },
+    body: jsonEncode(<String, String>{}),
+  );
+
+  print("net.dart body: " + response.body);
+
+  MasterData _masterData = MasterData.fromJson(json.decode(response.body));
+
+  print("ZZZZ: " + _masterData.usersToJson());
+
+  if (response.statusCode == 200) {
+    return _masterData;
+  } else {
+    return _masterData;
+  }
```

Comment:
Condition is not needed, i.o.w. does not result in different outcome.
Please remove this.